services/api-server/src/simcore_service_api_server/models/schemas/jobs.py

```diff
@@ -232,7 +232,7 @@ class JobStatus(BaseModel):
 
     job_id: UUID
     state: TaskStates
-    progress: PercentageInt = Field(default=0)
+    progress: PercentageInt = Field(default=0)  # type: ignore
```

Comment:
```suggestion
    progress: PercentageInt = Field(default=PercentageInt(0))
```
This would also work. And I think this does not parse in this case. maybe to double check

Reply:
> This would also work. And I think this does not parse in this case. maybe to double check

mmmh I do not like that solution.  PercentageInt **is** an int so it should not complain. I think we are missing some tuning in the pydantic mypy plugin. Let's check that omorrow

Reply:
well. it's an int with some constraints. so technically it's less than an int. so I agree that 0 is part of what PercentageInt is supposed to be, but that would not be sufficient.